### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Esteban Lopez

### DISCUSSION THREAD

**Discussion by EstebanLopezGomez** [heig-vd-dai-course/heig-vd-dai-course#65] [Show and tell]
Practical Work 1

**Comment by EstebanLopezGomez** [comment]
Here's the link for my repository : https://github.com/EstebanLopezGomez/DAI_Practical_work_1

**Comment by ludelafo** [comment]
Could you please tell us a bit about what you plan to do for this practical work? :)

**Reply by EstebanLopezGomez** [reply]
Hello! I  was thinking to implement a dictionnary system to my CLI. This means that the user add a file which will be read by the CLI and it will count how many occurences of a word there are and to sort them alphabetically and show that in the output file.
